www/js/goals.js

```diff
@@ -591,7 +591,8 @@ angular.module('emission.main.goals',['emission.services', 'ngSanitize', 'ngAnim
       $rootScope.$on('$cordovaInAppBrowser:exit', function(e, event) {
         console.log("exiting, event = "+JSON.stringify(event));
       });
-  	}
+      $scope.refreshPage();
```

Comment:
Line 584 above, where I check to see if we are navigating to bic2cal is the one that triggers the close of the inAppBrowser, which in turn generates the exit event. If you want to refresh the page on exit, you need to put it inside the `$on`